### PR TITLE
Add `save-all` icon

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

File: icons/save-all.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "floppy disk"
```
Comment (karsa-mistmere) [inline icons/save-all.json:4]:
```suggestion
    "floppy disks",
    "copy"
```

**Review by karsa-mistmere — COMMENTED** [review]
Niiiiice, I think this could very well also stand in for infamous `save-as` as #119 #645

**Review by karsa-mistmere — APPROVED** [review]

File: icons/save-all.svg [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M20.2 18H7.8c-1 0-1.8-.8-1.8-1.8V3.8C6 2.8 6.8 2 7.8 2H18l4 4v10.2c0 1-.8 1.8-1.8 1.8z" />
+  <polyline points="18 18 18 11 10 11 10 18" />
+  <polyline points="10 2 10 6 16 6" />
+  <path d="M18 22H4c-1.1 0-2-.9-2-2V6" />
```
Comment (karsa-mistmere) [inline icons/save-all.svg:15]:
```suggestion
  <path d="M20.2 18a1.8 1.8 0 0 0 1.8-1.8V6l-4-4H7.8C6.8 2 6 2.8 6 3.8v12.4c0 1 .8 1.8 1.8 1.8Z" />
  <path d="M10 2v4h6" />
  <path d="M18 18v-7h-8v7" />
  <path d="M18 22H4a2 2 0 0 1-2-2V6" />
```

Comment (jguddas) [reply icons/save-all.svg:15]: To piggyback off of this one, I feel like a 2px radius fits better than the current 1.8.
```suggestion
  <path d="M6 4a2 2 0 0 1 2-2h10l4 4v10.2a2 2 0 0 1-2 1.8H8a2 2 0 0 1-2-2Z" />
  <path d="M10 2v4h6" />
  <path d="M18 18v-7h-8v7" />
  <path d="M18 22H4a2 2 0 0 1-2-2V6" />
```

**Comment by danielbayley** [comment]
> infamous `save-as` as defined in #119 #645

Oh I do like some of these though…

image

**Comment by jguddas** [comment]
I have this `file-convert` icon flying around as a `save-as` option , not 100% happy tho.

**Comment by danielbayley** [comment]
Well the existing `save` is a floppy disk, so `save-all` should be multiple of the same… But we could also have a more modern alternate as `save`[`-all`]`-2`…?

**Review by ericfennis — APPROVED** [review]

**Pull request merged**